scripts/build.py

```diff
@@ -462,9 +462,9 @@ def get_css_filepaths(dependency_bundle, dependency_dir):
 
     Args:
         dependency_bundle: dict(str, list(str) | str). The dict has three keys:
-            - 'js': List of paths to js files that need to be copied.
-            - 'css': List of paths to css files that need to be copied.
-            - 'fontsPath': Path to folder containing fonts that need to be
+            - 'js'. List of paths to js files that need to be copied.
```

Comment:
Should be followed by typeinfo: `'js': list(str). List`

Ditto below.

Reply:
Done